File: mongodb-nodejs-music/musicretrieveall.js
var MongoClient = require('mongodb').MongoClient;
var url = "mongodb://localhost";

MongoClient.connect(url, function(err, db) {
  if (err) throw err;
  var dbo = db.db("test");
  var query = {};
  dbo.collection("musicians").find(query).toArray(function(err, result) {
    if (err) throw err;
    console.log(result);
    db.close();
  });
});

File: mongodb-python-music/musicretrieveall.py
import pymongo

myclient = pymongo.MongoClient("mongodb://localhost/")
mydb = myclient["test"]
mycol = mydb["musicians"]

myquery = {"lastname":"Monk" }

mydoc = mycol.find(myquery)



for x in mydoc:
    print('hello')
#  print(x)